Project.toml

```diff
@@ -15,7 +15,7 @@ StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 
 [compat]
 DelimitedFiles = "1"
-GTPSA = "0.7.0"
+GTPSA = "0.7.0, 1"
 Reexport = "1"
 ReferenceFrameRotations = "3"
 SkewLinearAlgebra = "1"
```

Comment:
This is a test to see if I am automatically added as a reviewer if I make a comment